### ISSUE TITLE
Mix of .NET 7 dependencies

### ISSUE THREAD

**Issue by MaxMelcher** [Zimmergren/LogAnalytics.Client#49]
Hi Tobias,

I want to use your awesome library in a Azure Function (.net 6) project. 
I breaks in the latest version, 6.1.3, during runtime. It works in 6.1.2. 

There are now .NET 7 dependencies in the project and they might break the function runtime?
```
    <PackageReference Include="Microsoft.Extensions.Configuration.Abstractions" Version="7.0.0" />
    <PackageReference Include="Microsoft.Extensions.Options.ConfigurationExtensions" Version="7.0.0" />
    <PackageReference Include="Microsoft.Extensions.Http" Version="7.0.0" />
    <PackageReference Include="Microsoft.Extensions.Options" Version="7.0.0" />
```

Shouldn't they be version 6.x.x ? 

Thanks,
Max

**Comment by Zimmergren** [comment]
Hey @MaxMelcher , and thanks for pointing this out. 
You're right - they seem to have slipped in during the latest dependency update. Fixing now!

**Comment by Zimmergren** [comment]
Hey @MaxMelcher . 
There's a new version published that addressed this issue: `6.2.0`

- https://www.nuget.org/packages/LogAnalytics.Client/6.2.0

I hope this helps.
Tobias.
